### ISSUE TITLE
The new integration becomes the default editor for the bnd.bnd

### ISSUE THREAD

**Issue by juergen-albert** [eclipse-pde/eclipse.pde#672]
It is great for PDE Projects, but this should not be the case if the project has no pde nature. Right now, it becomes the default editor for all our bnd projects.

Can this be changed?

**Comment by merks** [comment]
Isn't this configurable?

![image](https://github.com/eclipse-pde/eclipse.pde/assets/208716/c01fddf4-3ed3-4015-b231-752cace2ba00)

**Comment by maho7791** [comment]
I already tried this, but it didn't work. I also cannot remove the Plugin-Manifest Editor entry in the preferences (like in your screenshot)

When I am in the bndtools perspective, I still get the PDE editor as default. It seems that the PDE is always taken as default.

**Comment by juergen-albert** [comment]
I found, that I have to select the Bnd Project Editor in the Associated Editor View and mark it as default editor again, in order to work.

**Comment by merks** [comment]
I would have expected that if you make a certain editor the default, then it must be the default.    So I tried that in my workspace

![image](https://github.com/eclipse-pde/eclipse.pde/assets/208716/0caff435-1e10-4e69-923a-db5e57908c70)

And I see even the icon changes afterwards:

![image](https://github.com/eclipse-pde/eclipse.pde/assets/208716/0a26c786-5f8f-484e-82c4-11009e89290a)

And double click opens BND editor.  So I think it does work.  But you must make it the default editor pushing the Default button...

**Comment by merks** [comment]
If you have Oomph installed, you could configure that automatically:
```
<?xml version="1.0" encoding="UTF-8"?>
<workbench:FileAssociationsTask
    xmi:version="2.0"
    xmlns:xmi="http://www.omg.org/XMI"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:workbench="http://www.eclipse.org/oomph/setup/workbench/1.0"
    xsi:schemaLocation="http://www.eclipse.org/oomph/setup/workbench/1.0 https://raw.githubusercontent.com/eclipse-oomph/oomph/master/setups/models/Workbench.ecore">
  <mapping filePattern="bnd.bnd"
      defaultEditorID="bndtools.bndProjectEditor"/>
</workbench:FileAssociationsTask>

```
You could use Navigate -> Open Setup -> User and copy that task into your user.setup so that this is the default everywhere...

Other than this, I'm not sure what you expect PDE to do about this?  It's reasonable that it registers its editor in the plugin.xml, right?

**Comment by peterkir** [comment]
> If you have Oomph installed, you could configure that automatically:
> 
> ```
> <?xml version="1.0" encoding="UTF-8"?>
> <workbench:FileAssociationsTask
>     xmi:version="2.0"
>     xmlns:xmi="http://www.omg.org/XMI"
>     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
>     xmlns:workbench="http://www.eclipse.org/oomph/setup/workbench/1.0"
>     xsi:schemaLocation="http://www.eclipse.org/oomph/setup/workbench/1.0 https://raw.githubusercontent.com/eclipse-oomph/oomph/master/setups/models/Workbench.ecore">
>   <mapping filePattern="bnd.bnd"
>       defaultEditorID="bndtools.bndProjectEditor"/>
> </workbench:FileAssociationsTask>
> ```
> 
> You could use Navigate -> Open Setup -> User and copy that task into your user.setup so that this is the default everywhere...
> 
> Other than this, I'm not sure what you expect PDE to do about this? It's reasonable that it registers its editor in the plugin.xml, right?

Yes, we are using [oomph setups](https://bndtools.org/bndtools.p2.repo/) and will add this, BUT
will this be added to an existing setting or replace all file associations?
